ci/run_test.sh

```diff
@@ -19,7 +19,9 @@ else
   BUILD_UI_OPT=${WITHOUT_BUILD_UI}
 fi
 
+mkdir jacoco-all
 # It's enough to build UI either with H2 or PostgreSQL not with the both
 "${BASEDIR}/run_test_h2.sh" ${WITHOUT_BUILD_UI}
+cp digdag-*/build/jacoco/*.exec jacoco-all/
```

Comment:
This line can be removed?

Reply:
Ah, `digdag-*/build/jacoco` is cleaned up per every testing?

Reply:
Yes. build directory is removed when `clean` task executed.